cooltools/lib/common.py

```diff
@@ -67,3 +72,140 @@ def assign_regions_to_bins(bin_ids, regions_span):
     region_ids[mask_assigned] = regions_span["name"][bin_reg_idx_lo[mask_assigned]]
 
     return region_ids
+
+
+def download_test_data(name="all", cache=True, data_dir=None):
```

Comment:
Maybe call it "example" data instead of "test" data? or maybe just `download_dataset`?